src/unitycatalog/test_utils/__init__.py

```diff
@@ -0,0 +1,2 @@
+from unitycatalog.test_utils.client_utils import *
+from unitycatalog.test_utils.function_utils import *
```

Comment:
can we avoid `*` import?

Reply:
I'd consider using `*` if I need to import 1000 items.

Reply:
That's true, but what's the concern of using `*`?

Reply:
https://stackoverflow.com/a/2386740 explains why

Reply:
what if client_utils and function_utils have the same function?

Reply:
Make sense :) Updated